tendrl/monitoring_integration/webhook/webhook_receiver.py

```diff
@@ -1,85 +1,69 @@
-import json
+import cherrypy
+from flask import Flask
+from flask import request
+from paste.translogger import TransLogger
```

Comment:
these are third party libraries, keep them in a separate group and not with stdlib